partcad-cli/src/partcad_cli/click/commands/render.py

```
@@ -0,0 +1,137 @@
+            packages = list(
+                map(
+                    lambda p: p["name"],
+                    list(all_packages),
+                )
+            )
+        else:
+            packages = [package]
+
+        for package in packages:
+            if not object is None:
+                if not ":" in object:
```

Ruff (0.8.2)

Test for membership should be `not in`

Convert to `not in`

(E713)